test/podmontest/insi.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin
+#
+# Copyright (c) 2021. Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+#
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+instances=${instances:-4}
+ndevices=${ndevices:-0}
+nvolumes=${nvolumes:-4}
+zone=${zone:-""}
+storageClassName=${storageClassName:-isilon}
+image="$REGISTRY_HOST:$REGISTRY_PORT/podmontest:v0.0.54"
+prefix="pmti"
+replicas=1
+deploymentType="statefulset"
+driverLabel="csi-isilon"
+podAffinity="false"
+unreachableTolerationSeconds=300
+
+if [ "$DEBUG"x != "x" ]; then
+  DEBUG="--dry-run --debug"
+fi
+
+for param in $*
+do
+    case $param in
+       "--instances")
+          shift
+          instances=$1
+          shift
+          ;;
+       "--ndevices")
```

Comment:
nit isilon doesn't support block devices so could remove this if you wanted

Reply:
Tests were performed with this change. Will remove this in the subsequent PR's. We still have few more changes to be pushed. Will keep note of this